app/services/document_capture_session_result.rb

```diff
@@ -8,9 +8,10 @@
   :attention_with_barcode,
   :failed_front_image_fingerprints,
   :failed_back_image_fingerprints,
+  :captured_at,
```

Comment:
Is there any issue with existing document_capture_sessions in redis not having this attribute?

Reply:
great catch ... added  dig(:captured_at) to check if it exists before evaluating 👍🏿